03.bowling/bowling.rb

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env ruby
+require 'rubygems'
+
+obtained_score = ARGV[0]
+scores = obtained_score.split(',')
+shots = []
+
+scores.each do |score|
+  if score == 'X'
+    shots << 10
+    shots << 0
+  else
+    shots << score.to_i
+  end
+end
+
+frames = []
+shots.each_slice(2) do |shot|
+  frames << shot
+end
+frames << [0, 0] if frames.size == 10
+
+point = 0
+frames.each_cons(2).with_index do |(current_frame, next_frame), i|
```

Comment:
each_consだと次のフレームは`next_frame`なのに次の次は`frames[i + 2]`になってしまうのでちょっと中途半端な印象です。
なのでただのeach_with_indexでもいいかもしれません。

```suggestion
frames.each_with_index do |current_frame, i|
  next_frame = frames[i + 1]
  after_next_frame = frames[i + 2]
```